api/src/test/java/org/openmrs/util/OpenmrsUtilTest.java

```diff
@@ -422,6 +424,33 @@ public void containsDigit_shouldReturnFalseIfStringContainsNoDigits() {
 		Assert.assertFalse(OpenmrsUtil.containsDigit(null));
 	}
 	
+
+	/**
+	 * @see OpenmrsUtil#containsonlyuppercase(String)
+	 */
+	@Test
+	public void containsUpperCase_shouldReturnIfStringContainsOnlyUpperCase() {
+
+		Assert.assertFalse(
+				OpenmrsUtil.containsUpperCaseOnly("this is the case"));
+
```

Comment:
remove the unnecesary space here
as advised here  https://wiki.openmrs.org/display/docs/Java+Conventions